modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -55,20 +56,23 @@ Duplicated copies of your specifications can be created on your local drive. You
 
 == API Specifications as API Specification Projects in Studio
 
-To start an API specification project in Studio and sync it with Design Center, see xref:create-api-specification-studio.adoc[]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and Design Center.
+To start an API specification project in Studio and sync it with MuleSoft VCS, see xref:create-api-specification-studio.adoc[]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and MuleSoft VCS.
 
-Import an API specification from Design Center as an API specification project with VCS-enabled capabilities to edit your API specification using Studio and sync your changes with Design Center:
+Import an API specification from MuleSoft VCS as an API specification project with VCS-enabled capabilities to edit your API specification using Studio and sync your changes with MuleSoft VCS:
 
-. Design an API specification in Design Center. +
+. Design an API specification in Design Center. 
++
 You can either create and publish an API specification with the xref:design-center::design-create-publish-api-visual-editor.adoc[Visual Editor] or with the xref:design-center::design-create-publish-api-raml-editor.adoc[Text Editor].
-. Import the API specification from Design Center into Studio. +
-You can import the API specification as an editable asset by xref:sync-api-projects-design-center.adoc[Synchronizing an API Specifications with Design Center].
+. Import the API specification from Design Center through MuleSoft VCS into Studio. 
```

Comment:
Should we change the `from Design Center` bit in here to `from Anypoint Platform`?

Reply:
I like it! Thanks!